Datos del coneval

nepito opened the pull request.

Commit:
Agrega los resultados al reporte

Commit:
Prueba que calcula el gini para todos los estados

Commit:
Obten la longitud de la salida buscada

Commit:
Prueba que es un dataframe con 33 renglones

Commit:
Haz que la salida sea un tibble de 33 renglones

Commit:
Prueba que tiene dos columnas

Commit:
Agrega dos columnas

Commit:
Prueba que una columna tiene la entidad federativa

Commit:
Implementa la función

Commit:
Actualiza la descripción de los resultados

Commit:
Cambia de coeficiente al índice de Gini

davidmacer commented on coneval/R/gini_by_state.R:
```diff
@@ -8,8 +8,15 @@ gini_by_state <- function(data, state){
         ineq::ineq(type="Gini")
 }
 
+
```
```suggestion
# Función que calcula el índice de Gini para cada estado
```

davidmacer commented on coneval/R/gini_by_state.R:
```diff
@@ -8,8 +8,15 @@ gini_by_state <- function(data, state){
+        dplyr::group_by(entidad_federativa) %>%
+        dplyr::summarize(gini = ineq::ineq(ingreso, type="Gini"))
+}
+
```
```suggestion
# Función que agrupa los datos del índice de gini para cada estado por decil
```

davidmacer commented on coneval/tests/testthat/test_read_gini_data.R:
```diff
@@ -28,4 +28,18 @@ test_that("gini_sonora has the expected shape", {
   expect_equal(obtained_gini, expected_gini, tolerance=1e-3)
+})
+
+
```
Aunque después de echarle un ojo al código y ver que está claro, no estaría de más poner un comentario breve sobre qué hace esta prueba; una línea debería ser suficiente.

nepito replied:
Excelente @davidmacer agregué las sugerencias pasadas solo para ir desempolvando.

Commit:
⚕️ Apply suggestions from code review

Co-authored-by: David Martínez <jdavid.macer@gmail.com>

Commit:
🔧 Add recipe for gini nacional

Commit:
🔥 Delete unused variable

Commit:
🔧 Update recipe

CerdoFunk committed:
Create color_mex_map.py

Primera aproximación para generar un mapa de colores de México, la idea es utilizar distintas variables asociadas a estados para determinar el comportamiento en distintas regiones del país.

CerdoFunk committed:
Create color_map.py

Primera aproximación para generar un mapa de colores de México, la idea es utilizar distintas variables asociadas a estados para determinar el comportamiento en distintas regiones del país.

CerdoFunk committed:
Delete color_map.py

Me apendeje y lo cargue dos veces el mismo archivo.py con dos nombres distintos.

CerdoFunk committed:
Update color_mex_map.py

Agregue un par de lineas para seleccionar regiones especificas del archivo de Gini que me compartió el nepo. Parece que ya se colorean los estados acorde a los valores de ingresos del decil I.

CerdoFunk committed:
Update color_mex_map.py

Puse en comentario linea 22 y 15 que pintan el mapa con colores aleatorios. Y corregí un error de dedo en línea 24. Aún así, el mapa no parece estar acorde a los valores que corresponden a los estados, aún no se por qué.

Commit:
➕🐳 Add dependency

Commit:
🚚 Move file to directory src

Commit:
🐳 Update parent image

Commit:
👩🏿‍💻 Change input file
